README.md

```diff
@@ -29,27 +37,22 @@ or if you want to rebuild the images
 
 `docker-compose up --build -d`
 
-#### Start
-
-`docker-compose start`
-
-#### Restart
-
-`docker-compose restart [service]`
-
-#### Stop
+When you run `docker-compose up`, Docker Compose automatically combines `docker-compose.yml` and `docker-compose.override.yml`. You can check the `docker-compose.override.yml` file, and you'll see the frontend is running locally on port 8080.
 
-`docker-compose stop`
+In production, the frontend is hosted on Cloudflare Pages, not locally.
 
-#### Down
+#### Other commands
 
-`docker-compose down`
+- `docker-compose start`
+- `docker-compose restart [service]`
+- `docker-compose stop`
+- `docker-compose down`
 
-## Setup
+## Setup Sandbox
 
 ### Sandbox
 
-1. in `.config/submission.json`:
-  - set `working_dir` to your desired location
+1. in `Sandbox/.config/submission.json`:
+  - set `working_dir` to your desired location to store the user's submission
 2. cd to `Sandbox` folder, run `./build.sh`, this will build the images you need to compile and execute user's submission
 3. that's all :P, i think
```

Comment:
I think those instructions can be moved before the Run Docker section as they should be done before that, and we should tell users they can check README in submodules to get more details about project configuration.

Reply:
Done